examples/dgmulti_2d/elixir_euler_brown_minion_vortex.jl

```diff
@@ -48,6 +48,6 @@ callbacks = CallbackSet(summary_callback, analysis_callback, alive_callback)
 
 tol = 1.0e-8
 sol = solve(ode, RDPK3SpFSAL49(), abstol=tol, reltol=tol,
-            save_everystep=false, callback=callbacks);
+            save_everystep=false, callback=callbacks, maxiters=typemax(Int));
```

Comment:
What should we use as default `maxiters` here?

Reply:
The default is `10^5`, see https://diffeq.sciml.ai/latest/basics/common_solver_opts/#Miscellaneous. However, we should think about automatic this instead of requiring people to add this keyword argument in every elixir - we will forget and make your effort fruitless in the end...

Reply:
To achieve that, we will probably need to modify `trixi_include` to find a call to `solve`, check whether it has a keyword argument `maxiters`, add it if it's not found with default value, and proceed as usual with the adaption of kwargs passed to `trxi_include`.

Reply:
That sounds like a good idea, but I'm afraid it's a bit out of my scope. How would I do that?

Reply:
#963

Reply:
You can use this new functionality when merging current `main` into this branch. Then, you should revert the additions of `maxiters` to `solve` calls.

Reply:
Already done locally, I'm just waiting for CI to see if coveralls is working now. Thanks for this!